.github/workflows/ci.yaml

```diff
@@ -0,0 +1,50 @@
+name: CI Testing
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        os: [ubuntu-latest]
+        include:
+          - name: Test on MacOS-X
+            os: macos-latest
+            python: 3.9
+          - name: Test on Windows
+            os: windows-latest
+            python: 3.9
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade setuptools pip wheel
+        python -m pip install black codecov
+        if [ -f requirements.txt ]; then python -m pip install -r requirements.txt; fi
```

Comment:
That if statement is pretty pointless and can be removed.

Reply:
Actually, I would remove that line altogether, since `requirements.txt` does not get used when installing from PyPI. Would make it a bit more realistic.